### PR TITLE
fix bugs in webportal job submission form

### PR DESCRIPTION
**Pull request by Asakuri**

### CODE REVIEW

File: webportal/src/app/job/job-submit/job-submit.component.js [hunk 1]
```diff
@@ -74,14 +74,16 @@ const exportFile = (data, filename, type) => {
 };
 
 const submitJob = (jobConfig) => {
+  console.log(jobConfig);
```
Comment (hwuu) [inline webportal/src/app/job/job-submit/job-submit.component.js]: Remove  this `console.log(...)`?

File: webportal/src/app/job/job-submit/job-submit.component.js [hunk 2]
```diff
@@ -112,20 +115,24 @@ const loadEditor = () => {
...
+  document.getElementById("editor-holder").style.height = heights - 300 + "px";
 };
 
 $('#sidebar-menu--submit-job').addClass('active');
 
 $('#content-wrapper').html(jobSubmitHtml);
 $(document).ready(() => {
   loadEditor();
+  editor.on('change', () => {
+    document.getElementById('submitJob').disabled = (editor.validate().length != 0);
```
Comment (hwuu) [inline webportal/src/app/job/job-submit/job-submit.component.js]: Use jQuery style -- `$('#submitJob').disabled = ...`.

File: webportal/src/app/job/job-submit/job-submit.schema.js [hunk 3]
```diff
@@ -98,7 +98,7 @@ const taskRoleSchema = {
       format: "textarea",
       propertyOrder: 1001,
       options: {
-        expand_height: true,
+        // expand_height: true,
```
Comment (hwuu) [inline webportal/src/app/job/job-submit/job-submit.schema.js]: What is the purpose?

Comment (Asakuri) [reply webportal/src/app/job/job-submit/job-submit.schema.js]: if the option is set to true, the editor should expand textarea automatically, but sometimes it doesn't work, so I explicitly set the style in css file.

**Pull request merged**